_posts/2021-04-12-ugrc-ng911-update.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 'UGRC Next-Generation 911 Update'
+author:
+  display_name: Erik Neemann
+  email: eneemann@utah.gov
+date: 2021-04-12 08:30:12

```

Comment:
Did you mean to set this to a future date? If we were to merge this today, it would skip this post because it's a date in the future. Then it wouldn't get published until another build is triggered (usually with a PR) after this date. I would suggest putting today's date.

Reply:
Ah, okay.  We can post today if you're ready to review/approve.  I was assuming it wouldn't be reviewed until Monday.  I'll update the date.